Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -33,10 +35,39 @@ abbrev LaurentSeries (R : Type*) [Zero R] :=
   HahnSeries ℤ R
 #align laurent_series LaurentSeries
 
-variable {R : Type u}
+variable {R : Type*}
 
 namespace LaurentSeries
 
+section Zero
+
+variable [Zero R]
+
+-- generalize to any locally finite linear order? see mathlib3 PR #18604
+theorem supp_bdd_below_supp_Pwo (f : ℤ → R) (n : ℤ) (hn : ∀(m : ℤ), m < n → f m = 0) :
+    (Function.support f).IsPwo := by
```

Comment:
Please merge `master`, then fix lower/upper cases: `IsPwo` was renamed to `IsPWO` and `IsWf` to `IsWF`.

Reply:
I put a Set.bddBelow_wellFoundedOn_lt result in Mathlib.Order.WellFoundedSet.  This can be combined with Set.isWF_iff_isPWO.